security-analysis/src/main/java/com/powsybl/security/SecurityAnalysisParameters.java

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2018, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security;
+
+import com.powsybl.commons.config.PlatformConfig;
+import com.powsybl.commons.extensions.AbstractExtendable;
+import com.powsybl.loadflow.LoadFlowParameters;
+
+import java.util.Objects;
+import java.util.ServiceLoader;
+
+/**
+ * Parameters for security analysis computation.
+ * Extensions may be added, for instance for implementation-specific parameters.
+ *
+ * @author Teofil Calin BANC <teofil-calin.banc at rte-france.com>
+ * @author Sylvain LECLERC <sylvain.leclerc@rte-france.com>
+ */
+public class SecurityAnalysisParameters extends AbstractExtendable<SecurityAnalysisParameters> {
+
+    private LoadFlowParameters loadFlowParameters = new LoadFlowParameters();
+
+    /**
+     * Load parameters from platform default config.
+     */
+    public static SecurityAnalysisParameters load() {
+        return load(PlatformConfig.defaultConfig());
+    }
+
+    /**
+     * Load parameters from a provided platform config.
+     */
+    public static SecurityAnalysisParameters load(PlatformConfig platformConfig) {
+        Objects.requireNonNull(platformConfig);
+
+        SecurityAnalysisParameters parameters = new SecurityAnalysisParameters();
+        parameters.readExtensions(platformConfig);
+
+        parameters.setLoadFlowParameters(LoadFlowParameters.load(platformConfig));
+
+        return parameters;
+    }
+
+    protected void readExtensions(PlatformConfig platformConfig) {
```

Comment:
This method has to be moved to a ExtensionConfigLoaderProviders utility class as it's done with serializers (to do in the next refactor PR)